spec/lib/publisher_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'spec_helper'
+require 'gds_api/test_helpers/content_store'
+require 'gds_api/test_helpers/publishing_api'
+
+describe Publisher do
+  let(:contact) { build :contact, :with_related_contacts, title: 'Chief Milliners Office' }
+
+  context '#publish' do
+    context 'with links' do
+      let(:presenter) { ContactPresenter.new(contact) }
+
+      it 'calls put_content' do
+        expect(Publisher.client).to receive(:put_content)
+        .with(contact.content_id, presenter.payload)
```

Comment:
According to the styleguide as enforced by [govuk-lint](https://github.com/alphagov/govuk-lint), these should be indented. It's worth adding govuk-lint to this repo, all bar two of the 274 contraventions can be automatically corrected. See the README for implementation.